packages/datadog-plugin-jest/src/index.js

```diff
@@ -218,6 +231,153 @@ function createHandleTestEvent (tracer, testEnvironmentMetadata, instrumenter) {
   }
 }
 
+function createWrapIt (tracer, globalConfig, globalInput, testEnvironmentMetadata) {
```

Comment:
At this point the plugin should be split in multiple files if many unrelated sub-packages are instrumented.

Reply:
yeah I agree

Reply:
I did just that: now we have `jest-environment.js` for the environment instrumentation and a separate `jest-jasmine2.js` for the jest jasmine instrumentation.